calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -327,35 +321,35 @@ kubectl replace -f rule set.configmap.yaml
 
 Read more about the order of execution for plugins here: https://coreruleset.org/docs/concepts/plugins/
 
-### View WAF events
+## View WAF events
 
-#### Security Events
+### Security Events
 
 To view WAF events in a centralized security events dashboard, go to: **Threat defense**, **Security Events**. For help, see [Security Event Management](../threat/security-event-management).
 
-#### Kibana
+### Kibana
 
 To view WAF events In Kibana, select the `tigera_secure_ee_waf*` index pattern.
 
-#### Disable WAF for a service
+### Disable WAF for a deployment
 
-To disable WAF on a service, use the Actions menu on the WAF board, or use the following command:
+To disable WAF on a deployment, use the Actions menu on the WAF board, or use the following command:
 
 ```bash
-kubectl annotate svc <service-name> -n <service-namespace> projectcalico.org/l7-logging-
+   kubectl patch deployment <deployment-name> -n <deployment-namespace> --type='json' -p '[{"op":"remove","path":"/spec/template/metadata/labels/applicationlayer.projectcalico.org~1sidecar"},{"op":"remove","path":"/spec/template/metadata/annotations/applicationlayer.projectcalico.org~1waf"}]'
```

Comment:
@radixo does this look right for the disable command for WAF?

Reply:
Actually it is, the bad thing I left behind were the unnecessary spaces on the left, you can trim them. But about the `~1` string, is how you can scape the `/` on json paths.